libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -626,8 +628,11 @@ void AP_GPS::update_instance(uint8_t instance)
             state[instance].hdop = GPS_UNKNOWN_DOP;
             state[instance].vdop = GPS_UNKNOWN_DOP;
             timing[instance].last_message_time_ms = tnow;
+            timing[instance].delta_time_ms = GPS_TIMEOUT_MS;
         }
     } else {
+        // the delta time will be lost until we have parsed two messages, clamp it to prevent excessive growth
```

Comment:
I still don't like the message, let me try to give an alternative: *delta will only be correct after parsing two messages* or *the first delta time won't match reality, only works after parsing two messages*.